client/state/reader/follows/schema.js

```diff
@@ -0,0 +1,22 @@
+export const items = {
+	type: 'object',
+	patternProperties: {
+		//be careful to escape regexes properly
+		'.+': {
+			type: 'object',
+			required: [ 'feed_URL', 'URL', 'is_following' ],
+			properties: {
+				URL: { type: 'string' },
+				feed_URL: { type: 'string' },
+				is_following: { type: 'boolean' },
+				ID: { type: [ 'integer', 'null' ] },
+				blog_ID: { type: [ 'integer', 'null' ] },
```

Comment:
so there are two cases:
from sync, it'll just be 0
from a follow of a URL, it'll be missing (for now)